checker/tests/nullness-extra/issue607/Makefile

```diff
@@ -0,0 +1,9 @@
+.PHONY: all
+
+all: clean
+	$(JAVAC) -source 8 -target 8 Issue607SuperClass.java Issue607Interface.java
```

Comment:
Is the `-source 8 -target 8` really necessary here? Isn't that the default for the `JAVAC` that's being used here?